bugsnag/django/__init__.py

```diff
@@ -45,14 +46,23 @@ def add_django_request_to_notification(notification):
 
     if getattr(request, "session", None):
         notification.add_tab("session", dict(request.session))
-    notification.add_tab("request", {
+    req = {
         'method': request.method,
         'path': request.path,
         'encoding': request.encoding,
         'GET': dict(request.GET),
         'POST': dict(request.POST),
         'url': request.build_absolute_uri(),
-    })
+    }
+    if 'CONTENT_TYPE' in request.META:
+        if request.META['CONTENT_TYPE'] == 'application/json':
```

Comment:
This could be simplified a bit:

```suggestion
    if request.META.get('CONTENT_TYPE') == 'application/json':
```

Reply:
Fixed in 4b59d6cf5ee86e11ede693f10421c1f95379d044